ntc_templates/templates/cisco_ios_show_power_inline.textfsm

```diff
@@ -0,0 +1,28 @@
+Value Required INTERFACE (\S+)
+Value ADMIN (auto)
```

Comment:
Is this the only value ADMIN can have ?  (auto)

Reply:
I have access to 434 Cisco IOS switches and (auto) is the only value observed. Probably other values are possible, but I cannot confirm or test.